launch/org.eclipse.cdt.docker.launcher/src/org/eclipse/cdt/docker/launcher/ContainerCommandLauncher.java

```diff
@@ -308,6 +308,12 @@ public Process execute(IPath commandPath, String[] args, String[] env, IPath wor
 		additionalDirs.addAll(additionalPaths.stream().map(p -> ContainerLaunchUtils.toDockerVolume(pathMap, p))
 				.collect(Collectors.toList()));
 
+		final var getImg = ContainerLaunchUtils.provideDockerImage(monitor, connectionName, imageName);
```

Comment:
As mentioned below, this can return an IStatus and then it makes sense what you do with it.